specification/resources/resource-manager/Microsoft.Resources/stable/2020-10-01/subscriptions.json

```diff
@@ -0,0 +1,730 @@
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "x-ms-examples": {
+          "Get locations with a subscription id": {
+            "$ref": "./examples/GetLocations.json"
+          },
+          "Get locations with extended locations": {
+            "$ref": "./examples/GetLocationsWithExtendedLocations.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}": {
+      "get": {
+        "tags": [
+          "Subscriptions"
+        ],
+        "operationId": "Subscriptions_Get",
+        "description": "Gets details about a specified subscription.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns information about the subscription.",
+            "schema": {
+              "$ref": "#/definitions/Subscription"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get a single subscription.": {
+            "$ref": "./examples/GetSubscription.json"
+          }
+        }
+      }
+    },
+    "/subscriptions": {
+      "get": {
+        "tags": [
+          "Subscriptions"
+        ],
+        "operationId": "Subscriptions_List",
+        "description": "Gets all subscriptions for a tenant.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns an array of subscriptions.",
+            "schema": {
+              "$ref": "#/definitions/SubscriptionListResult"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get all subscriptions.": {
+            "$ref": "./examples/GetSubscriptions.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/tenants": {
+      "get": {
+        "tags": [
+          "Tenants"
+        ],
+        "operationId": "Tenants_List",
+        "description": "Gets the tenants for your account.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns an array of tenants.",
+            "schema": {
+              "$ref": "#/definitions/TenantListResult"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get all tenants.": {
+            "$ref": "./examples/GetTenants.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.Resources/checkResourceName": {
+      "post": {
+        "tags": [
+          "Tenants"
+        ],
+        "operationId": "checkResourceName",
```

Comment:
OperationId must be of the form "OperationGroup_Operation", like "Resources_CheckName", or "Tenants_CheckResourceName"

Reply:
These are copied from the previous version. Can we leave them? BTW, the first commit of this PR was a copy from previous version, and the subsequent commits contain the actual changes.

Reply:
yes, this will just add it as a top level client method